apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/behandling/Behandling.kt

```diff
@@ -89,9 +89,9 @@ fun Grunnlag.mapSoeker(): Soeker = with(this.soeker) {
     val navn = hentNavn()!!.verdi
 
     Soeker(
-        fornavn = navn.fornavn,
-        mellomnavn = navn.mellomnavn,
-        etternavn = navn.etternavn,
+        fornavn = navn.fornavn.capitalize(),
```

Comment:
Ser at capitalize er deprecated i Kotlin. `@Deprecated("Use replaceFirstChar instead.", ReplaceWith("replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }", "java.util.Locale"))
`

Som er ei litt klønete erstatning, men sånn er det. Kan jo ha vår eigen extension function som gjer akkurat det?
navn.fornavn.storForbokstav() eller noko

Reply:
Ja, hvorfor tror du jeg laget min egen bare noen få linjer ned herfra (L137) ?